<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoissonImageSynthesisClass</class>
 <widget class="QWidget" name="PoissonImageSynthesisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PoissonImageSynthesis</string>
  </property>
  <widget class="QLabel" name="labelBase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>390</width>
     <height>270</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlend">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>390</width>
     <height>270</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMix">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>390</width>
     <height>270</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>390</width>
     <height>270</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>98</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ベース画像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>98</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ブレンド画像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>98</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mixing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>98</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Importing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iterateButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Iterate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton_1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set 4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>反転</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SORSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>1.900000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>820</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SOR係数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set 5</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="poissonimagesynthesis.qrc"/>
 </resources>
 <connections/>
</ui>
